README.md

```diff
@@ -73,126 +89,161 @@ bash ./scripts/tests/register_mint_quantity.sh
-* [`iroha_permissions_validators`](permissions_validators) — permission validation logic
-* [`iroha_substrate`](substrate) — bridge substrate `XClaim` external module
-* [`iroha_telemetry`](telemetry) — monitoring and analysis of telemetry data
-* [`iroha_version`](version) — message versioning for non-simultaneous system updates
+* [`iroha`](cli) is the command-line application for deploying an Iroha peer. Contains the routing table and definitions of API endpoints.
+* [`iroha_actor`](actor) provides a message passing model for Iroha components.
+* [`iroha_client`](client) provides a library for building clients that communicate with peers.
+* [`iroha_client_cli`](client_cli) is the reference implementation of a client.
+* [`iroha_config`](config) handles configuration and documentation generation for options and run-time changes.
+* [`iroha_core`](core) is the primary library used by all other crates, including the peer endpoint management.
+* [`iroha_crypto`](crypto) defines cryptographic aspects of Iroha.
+* [`kagami`](tools/kagami) is used to generate cryptographic keys, default genesis, configuration reference, and schema.
+* [`iroha_data_model`](data_model) defines common data models in Iroha.
+* [`iroha_futures`](futures) is used for `async` programming.
+* [`iroha_logger`](logger) uses `tracing` to provide logging facilities.
+* [`iroha_macro`](macro) provides the convenience macros.
+* [`iroha_p2p`](p2p) defines peer creation and handshake logic.
+* [`iroha_permissions_validators`](permissions_validators) defines permission validation logic.
+* [`iroha_substrate`](substrate) is the bridge substrate `XClaim` external module.
+* [`iroha_telemetry`](telemetry) is used for monitoring and analysis of telemetry data.
+* [`iroha_version`](version) provides message versioning for non-simultaneous system updates.
 
 # Maintenance
+
+A brief overview on how to configure and maintain an Iroha instance:
+
+- [Configuration](#configuration)
+- [Endpoints](#endpoints)
+- [Logging](#logging)
+- [Monitoring](#monitoring)
+- [Storage](#storage)
+- [Scalability](#scalability)
+
 ## Configuration
 
-A detailed breakdown of all available configuration parameters is available [here](./docs/source/references/config.md). All configuration parameters can be either provided as a `config.json` or using environment variables.
+You can provide configuration parameters either as a `config.json` or using environment variables. Refer to the [detailed list](./docs/source/references/config.md) of all available configuration parameters.
 
-It may be useful to generate the configurations by looking at [`core/src/samples.rs`](./core/src/samples.rs) and [`client/src/samples.rs`](./core/src/samples.rs) to see examples that can be serialised into `json` and used for your needs.
+Configuration examples you may use as a reference point: 
+- [`core/src/samples.rs`](./core/src/samples.rs)
+- [`client/src/samples.rs`](./core/src/samples.rs)
```

Comment:
@appetrosyan these links are broken, where should I refer to instead?

Reply:
Replace both with `cli/src/samples.rs`